docs/README.md

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 1
+sidebar_label: Introduction
+---
+
+# The Zui Desktop Application
+
+Zui (formerly called Brim) is a graphical user interface for exploring data in
+[Zed lakes](https://zed.brimdata.io/docs/commands/zed). While these docs are
+specific to the Zui app experience, you may want to browse the separate
+[docs for the Zed project](https://zed.brimdata.io/docs) on topics such as:
+
+* the [Zed language](https://zed.brimdata.io/docs/language) that's used in Zui for queries and analytics,
+* the [command-line tools](https://zed.brimdata.io/docs/commands) that work alongside Zui, and
+* the [Zed data formats](https://zed.brimdata.io/docs/formats).
+
+:::tip pcap processing
```

Comment:
I think this callout should be at the bottom of the page. It might be jarring for a first time user to see the words "zeek", "suricata" and "pcaps" so soon. They might think Zui is not relevant to them. 

But if were at the bottom after the info on the name change, it would seem appropriate.